RELEASENOTES/v2.0.0.rc-1.MD

```diff
@@ -0,0 +1,64 @@
+# Notary v2 Release Candidate 1 
+
+## Overview
+Notary v2 working group is pleased to announce its first release candidate, RC-1. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-core-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-rc.1)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.1)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-rc.1)](https://github.com/notaryproject/notation/releases/tag/v1.9.0-rc.1)***   (Updated in this release) 
+- [Notary v2 Specs](https://github.com/notaryproject/notaryproject) ***[(notaryproject-1.0.0-rc.1)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft.3)*** (Updated in this release)
```

Comment:
We can also consider to cut rc.1 release for specs, so the tag is
`https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-rc.1`

Reply:
Fixed.